bin/conjur-cli.rb

```diff
@@ -78,6 +78,13 @@ def test_select
       exec "rake authn_local:run"
     end
 
+    # Start the rotation "watcher" in a separate thread
+    rotations_thread = Thread.new do
+      exec "rake expiration:watch[#{account}]"
```

Comment:
That looks weird and it's not clear (to me) how it behaves re database connections for instance.
Would it work to just call `MasterRotator.new(rotators: InstalledRotators.new).rotate_every(1.second)` here instead?

Reply:
@dividedmind So changing from process to threads I _thought_ was your idea.  It may have been Kevin's.  I believe the new threads can share db connections and that was your reason for preferring them, but that might be incorrect.  

You are right to notice that using rake is sort of superfluous here... but I had to use to get the autoloading to work.  rake integrates with rails, and if I try to call `MasterRotator` directly in `conjur-cli.rb` I get a bunch of class not found errors, and that necessitates putting explicit require statements everywhere, which ironically is my preferred way of doing things, but it's very un-railsy and would be departing from the conventions used everywhere else in our codebase.  So rake is being used here for the sole purpose of hooking into the rails autoload system.  If you have a better solution I'm all ears.